source/fuzz/fuzzer_pass_apply_id_synonyms.cpp

```diff
@@ -85,6 +85,12 @@ void FuzzerPassApplyIdSynonyms::Apply() {
         auto synonym_to_try = synonyms_to_try[synonym_index];
         synonyms_to_try.erase(synonyms_to_try.begin() + synonym_index);
```

Comment:
Could you replace 84-86 with a single call to RemoveAtRandomIndex (so that the synonym_index temporary disappears)?